pkg/scheduler/scheduler.go

```diff
@@ -230,13 +232,16 @@ func (sched *Scheduler) preempt(preemptor *v1.Pod, scheduleErr error) (string, e
 			glog.Errorf("Error in preemption process. Cannot update pod %v annotations: %v", preemptor.Name, err)
 			return "", err
 		}
+		podsPreempted := 0
 		for _, victim := range victims {
 			if err := sched.config.PodPreemptor.DeletePod(victim); err != nil {
 				glog.Errorf("Error preempting pod %v/%v: %v", victim.Namespace, victim.Name, err)
 				return "", err
 			}
+			podsPreempted++
```

Comment:
I would define an increment counter with `PreemptedPodsCounter := prometheus.NewCounter( ... )`, so I only need to call `metrics.PreemptedPodsCounter.Inc()` here, which is cleaner.

No enforcement here, just a suggestion.

Reply:
Yeah but when I was going through documentation, I found that counters are used when we are sure that values always increase and never decrease. But there is a chance that PreemptedPodsCounter will decrease(when the pods preempted are less in current iteration). So didn't use it.

From documentation,  `Counters should not be used to expose current counts of items whose number can also go down, e.g. the number of currently running goroutines. Use gauges for this use case.`

Reply:
That would monotonically increase. That could be useful too, but I would like to know how many pods were preempted in each preemption attempt. So, the current implementation matches what I have in mind.